src/main/java/edu/kis/vh/nursery/IntArrayStack.java

```diff
@@ -0,0 +1,35 @@
+package edu.kis.vh.nursery;
+
+public class IntArrayStack {
+    public static final int MAX_LENGTH = 12;
+    public static final int EMPTY_VALUE = -1;
+    public static final int MAX_INDEX = 11;
```

Comment:
This constant is related to MAX_LENGTH. Changing only one leads to an error (inconsistency).